### DISCUSSION TITLE
Newbie Question

### DISCUSSION THREAD

**Discussion by elbatico1** [Acly/krita-ai-diffusion#430] [General]
Hi, thank you for the great work you've done so far.
How can I input a negative prompt? Is there some syntax that I have to use?
Thank you again for your work, and apologies if I'm wasting your time.

**Comment by nickthesail0r** [comment]
> Hi, thank you for the great work you've done so far. How can I input a negative prompt? Is there some syntax that I have to use? Thank you again for your work, and apologies if I'm wasting your time.

you press the configuration button, then interface, and turn on negative prompt

**Reply by elbatico1** [reply]
Thank you, now that I see it I feel very stupid.
Thank you!
